packages/myst-cli/src/utils/whiteLabelling.ts

```diff
@@ -0,0 +1,28 @@
+export function readableName(): string {
+  if ('MYSTMD_READABLE_NAME' in process.env) {
+    const name = process.env.MYSTMD_READABLE_NAME as string;
+    return `${name} (via myst)`;
+  } else {
+    return 'myst';
```

Comment:
This is a direct replacement:

```suggestion
    return 'MyST';
```